src/models/task.js

```diff
@@ -83,7 +82,7 @@ export default class Task {
 
 		if (!this.vtodo.hasProperty('uid')) {
 			console.debug('This task did not have a proper uid. Setting a new one for ', this)
-			this.vtodo.addPropertyWithValue('uid', uuid())
+			this.vtodo.addPropertyWithValue('uid', crypto.randomUUID())
```

Comment:
This function is supported by all Nextcloud-supported browsers:
* https://docs.nextcloud.com/server/latest/user_manual/sv/webinterface.html#web-browser-requirements
* https://browserslist.dev/?q=PjAuMjUlLCBub3Qgb3BfbWluaSBhbGwsIG5vdCBkZWFkLCBGaXJlZm94IEVTUg%3D%3D
* https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility

Reply:
Added a fallback for insecure contexts.